### PR TITLE
Add bulk action "add to an existing product model"

### PR DESCRIPTION
**Pull request by Delphineray**

### CODE REVIEW

File: content/md/products-super-power/product-mass-actions.md [hunk 1]
```diff
@@ -85,12 +85,36 @@ In order to save you some time, you can also directly categorize products in cat
...
 | Remove from categories | The products will be removed from the selected categories                                     |
 
 Confirm the action with the `Confirm` button located in the middle of the screen.
 
+# Add products to an existing product model
+
+To ease the enrichment of [products with variants](/articles/what-about-products-variants.html), you can gather products in a product model to allow the enrichment of their common properties.
+
+1.  Search for products in the grid with the appropriate filters
+2.  Select the products to add to an existing product model: Tick the products you want to add via the down arrow of the toolbar: `All` for all the products recorded, `All visible` for all visible products on the ongoing page, `none` for none of the products
```
Comment (KmilleF) [inline content/md/products-super-power/product-mass-actions.md]: is upper case required for "Tick"?

File: content/md/products-super-power/product-mass-actions.md [hunk 2]
```diff
@@ -85,12 +85,36 @@ In order to save you some time, you can also directly categorize products in cat
...
+3.  Click on `Bulk Actions`
+4.  Select `Add products to an existing product model` and then `Next`
+5.  Select a family, a [family variant](/articles/what-about-products-variants.html#what-is-a-family-variant) and a [product model](/articles/what-about-products-variants.html#what-is-a-product-model)
```
Comment (KmilleF) [inline content/md/products-super-power/product-mass-actions.md]: UI question: why didn't we call it "family variant" but "variant"?

File: content/md/products-super-power/product-mass-actions.md [hunk 3]
```diff
@@ -85,12 +85,36 @@ In order to save you some time, you can also directly categorize products in cat
...
+![Mass edit options](../img/Products_BulkActionAddProductModel.png)
+6.  Click on `Next`
+7.  Confirm the action with the `Confirm` button located in the middle of the screen
+8.  The products allowed will be added to the selected product model
+
+:::info
+The attributes defined as `common attributes` in the [family variant](/articles/what-about-products-variants.html#what-is-a-family-variant) will be now enriched in the product model, the previous values for these attributes on the products are removed. For example, if you add 2 products with an existing description and you would like to gather them in a product model and manage a common description, their previous descriptions are removed.
```
Comment (KmilleF) [inline content/md/products-super-power/product-mass-actions.md]: At 1st read I thought that attributes will be automatically enriched:). Maybe precised "at the product model level"?

File: content/md/products-super-power/product-mass-actions.md (hunk 1 above)
Comment (Delphineray) [reply content/md/products-super-power/product-mass-actions.md]: it as a copy/paste, I change it for "tick"

File: content/md/products-super-power/product-mass-actions.md (hunk 2 above)
Comment (Delphineray) [reply content/md/products-super-power/product-mass-actions.md]: In the UI, it's always displayed with a family context, often the family is before and Stéphane found it less heavy than always put '"family variant". In the documentation, I prefer to use "family variant" to have no confusion.

File: content/md/products-super-power/product-mass-actions.md (hunk 3 above)
Comment (Delphineray) [reply content/md/products-super-power/product-mass-actions.md]: ok

**Pull request merged**